### ISSUE TITLE
Unable to install arcgisbinding package offline

### ISSUE THREAD

**Issue by sujithperatla** [R-ArcGIS/r-bridge-install#72]
Please check the details below and help me..

Executing: RInstallDetails
Start Time: Thu Aug  2 10:36:57 2018
Running script RInstallDetails...
R (version 3.5.1), installed in: C:\Program Files\R\R-3.5.1
R packages will be installed into: C:\Users\peratla.sujith\Documents\R\win-library\3.5
All R package libraries detected: C:\Users\peratla.sujith\Documents\R\win-library\3.5;C:\Program Files\R\R-3.5.1\library
The ArcGIS R package is not installed. Use the 'Install R Bindings' tool to install it.
Completed script RInstallDetails...
Succeeded at Thu Aug  2 10:36:57 2018 (Elapsed Time: 0.51 seconds)

---------

Executing: InstallBindings false 3.5.1
Start Time: Thu Aug  2 10:44:40 2018
Running script InstallBindings...
Unable to access'https://api.github.com/repos/R-ArcGIS/r-bridge/releases/latest', error: [Errno 11002] getaddrinfo failed.
Unable to get current release information. Trying offline installation.
Found local copy of binding, installing from zip
Rcmd.exe INSTALL c:\users\peratl~1.suj\appdata\local\temp\arc8562\tmpevebic\arcgisbinding_1.0.1.232.zip
* installing to library 'C:/Users/peratla.sujith/Documents/R/win-library/3.5'
Warning in read.dcf(file.path(pkgname, "DESCRIPTION"), c("Package", "Type")) :
cannot open compressed file 'arcgisbinding/DESCRIPTION', probable reason 'No such file or directory'
Error in read.dcf(file.path(pkgname, "DESCRIPTION"), c("Package", "Type")) :
cannot open the connection
In R CMD INSTALL
R command returned non-zero exit status.
Rscript.exe c:\users\peratl~1.suj\appdata\local\temp\arc8562\tmpevebic\install.R
Installing package into 'C:/Users/peratla.sujith/Documents/R/win-library/3.5'
R command returned non-zero exit status.
Fallback installation method failed.
Completed script InstallBindings...
Succeeded at Thu Aug  2 10:44:59 2018 (Elapsed Time: 19.30 seconds)

**Comment by scw** [comment]
@sujithperatla it looks like the download is being corrupted. The arcgisbinding_1.0.1.232.zip file should have a size of 2306356 bytes, and a SHA-256 sum of 34176dfa0647b06b0417ca3c7e19b7f258d35eeb92b75939b840708dc830e52e.

**Comment by sujithperatla** [comment]
Thanks for the reply @scw 

Now I re-downloaded all files and started again as below:

Since I am unable to access https://api.github.com/repos/R-ArcGIS/r-bridge/releases/latest,
I'm following below steps :

1. Installed R (version 3.5.1) in C:\Program Files\R\R-3.5.1.
2. Downloaded r-bridge-install-master.zip (it showed 648 kb) from GIT and unzipped to C:\Users\peratla.sujith\Downloads folder.
3. Downloaded arcgisbinding_1.0.1.232.zip (it showed 2253 kb) from GIT.
4. To install ArcGIS R package, I added arcgisbinding_1.0.1.232.zip to C:\Users\peratla.sujith\Downloads\r-bridge-install-master folder.
5. Opened ArcMap (10.6) as Administrator - this is 32bit software
6. Added r-bridge-install-master folder to ArcMap (10.6) Catalog.
7. Run 'Install R bindings' tool.

Now I'm able to install ArcGIS R package. Thanks for your help
